src/ray/gcs/gcs_server/test/gcs_server_rpc_test.cc

```diff
@@ -725,12 +782,43 @@ TEST_F(GcsServerTest, TestErrorInfo) {
 }
 
 TEST_F(GcsServerTest, TestWorkerInfo) {
-  rpc::WorkerFailureData worker_failure_data;
-  worker_failure_data.mutable_worker_address()->set_ip_address("127.0.0.1");
-  worker_failure_data.mutable_worker_address()->set_port(5566);
+  // Report worker failure
```

Comment:
I think `ReportWorkerFailure` should only be done for existing workers (those have been added). We should check that in the GcsWorkerManager.